pkg/indexer/types.go

```diff
@@ -30,6 +30,16 @@ type status struct {
 	LedgerIndex uint32
 }
 
+type protocol struct {
+	ID uint `gorm:"primaryKey;notnull"`
+	// The version of the protocol running
+	Version byte
+	// The network name
+	NetworkName string
```

Comment:
Instead of creating the protocol table, you can simply add this to the status table.